packages/esm-patient-chart-app/src/actions-buttons/add-past-visit.component.tsx

```diff
@@ -2,16 +2,24 @@ import { showModal } from '@openmrs/esm-framework';
 import React, { useCallback } from 'react';
 import { useTranslation } from 'react-i18next';
 
-interface AddPastVisitOverflowMenuItemProps {}
+interface AddPastVisitOverflowMenuItemProps {
+  patientUuid?: string;
```

Comment:
`patientUuid` is optional?

Reply:
Yes, it is only needed by the patient search banner, but not the patient banner.